trogdor/src/test/java/org/apache/kafka/trogdor/common/WorkerUtilsTest.java

```diff
@@ -176,7 +176,7 @@ public void testCreatesNotExistingTopics() throws Throwable {
             new TopicDescription(
                 TEST_TOPIC, false,
                 Collections.singletonList(
-                    new TopicPartitionInfo(0, broker1, singleReplica, Collections.<Node>emptyList()))),
+                    new TopicPartitionInfo(0, broker1, singleReplica, Collections.<Node>emptyList(), Collections.<Node>emptyList(), Collections.<Node>emptyList()))),
```

Comment:
Pretty sure we don't need the type here. Java should be able to infer the type from the TopicPartitionInfo argument.